### DISCUSSION TITLE
Remove text within a rectangular region of page

### DISCUSSION THREAD

**Discussion by sh-l** [Hopding/pdf-lib#1180] [Q&A] [answered]
I'd like to remove texts at certain position of a page.
Can the library extract every "object", be it texts, or images, and then remove it?

pdf.js provides an api to extract text content. I'm not interested in text, just need to retrieve every single text that is rendered in certain region of a page.

**Comment by sh-l** [comment, marked as answer]
I ended up writing a Java program using [Apache PDFBox](https://github.com/apache/pdfbox). It seems to have an exhaustive feature set, and its repo has an example code to extract texts from a specified rectangular region.

**Reply by abrarhossain-bidgely** [reply]
I believe you are referring to  [PDFTextStripperByArea](https://javadoc.io/static/org.apache.pdfbox/pdfbox/2.0.29/org/apache/pdfbox/text/PDFTextStripperByArea.html) for getting text from a specific region. How did you remove it? Do you have any example code ?
